genie-core/src/main/java/com/netflix/genie/core/util/MetricsUtils.java

```diff
@@ -80,4 +83,44 @@ private MetricsUtils() {
     public static Map<String, String> newFailureTagsMapForException(final Throwable t) {
         return addFailureTagsWithException(Maps.newHashMap(), t);
     }
+
+    /**
+     * Convenience method to add common tags for slicing and dicing atlas metrics from Job
+     * execution environment to the map of tags passed in.
+     * Populates defaultIdValue in case ids are not present in the job execution environment
+     * E.g defaultIdValue will be added to the tags maps for the cluster_id key in case cluster_id
+     * does not exist in jobExecEnv
+     *
+     * @param tags The map of tags
```

Comment:
In intellij `cmd + option + L` auto formats the code and will space out these parameter documentation nicely. I only mention this cause I tend to do this habitually as I'm in files which leads to stupid modifications on files which really aren't code changes but formatting. Better to do it up front.